### ISSUE TITLE
Obsługa back-end'u 

### ISSUE THREAD

**Issue by DocentSzachista** [Droniu/emotion-visualisation#1]
POC:
- ładowanie danych przy starcie serwera i trenowanie PCA na danych które są zapisane na dysku
    - Do przemyślenia: wydobycie klasy rozpoznanej przez  sieć neuronowej z logitsów. Podajemy wartość maksymalną logitsa i index tej wartości tłumaczymy na klasę jaką rozpoznał. Jeżeli chcemy próg decyzyjny, musimy policzyć softmax'em to. 
    - Zastanowić się nad tym czy chcemy przy każdym ładowaniu back-endu uwzględniać wyliczanie tych danych czy też policzyć to gdzieś z boku i wstawić poprawiony zbiór danych. 

- Ładowanie modelu sieci neuronowej 
- Wystawienie endpointów na następujące rzeczy:
   - Dodawanie nowych tekstów do analizy (endpoint powinien zwrócić w odpowiedzi informację, czy tekst jest za długi bądź układ współrzędnych, + emocję z jaką się go powinno łączyć) 
   - Wystawianie listy dwuwymiarowej zawierającej układy współrzędne punktów już wykorzystanych do PCA.  
   - Może zwracanie listy logitsów, która została potraktowana funkcją softmax, tak aby dostać taką odpowiedź do wizualizacji:
   ```
   {
        "anger": 0.95,
        "love": 0.1,
        ....
   }
   ```
